### PR TITLE
[ADD] product_seeds: seed_package as delegate to product.attribute

### PR DESCRIPTION
**Pull request by robinkeunen**

### CODE REVIEW

File: product_seeds/data/data.xml [hunk 1]
```diff
@@ -1,6 +1,13 @@
 <?xml version="1.0" encoding="utf-8"?>
 <odoo>
-    <data noupdate="1">
+    <data noupdate="0">
```
Comment (houssine78) [inline product_seeds/data/data.xml:4]: if it's 0 then you can remove the attribute as it's the default value

File: product_seeds/models/seed_package.py [hunk 2]
```diff
@@ -0,0 +1,54 @@
+# © 2016 Robin Keunen, Coop IT Easy SCRL fs
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+
+from odoo import models, fields, api, _
+from odoo.exceptions import ValidationError
+
+
+class ProductAttribute(models.Model):
+    _inherit = 'product.attribute'
+    is_package = fields.Boolean()
```
Comment (houssine78) [inline product_seeds/models/seed_package.py:10]: pep8 says nothing? :)

File: product_seeds/models/seed_package.py [hunk 3]
```diff
@@ -0,0 +1,54 @@
...
+class ProductAttributeValue(models.Model):
+    _inherit = 'product.attribute.value'
+
+    @api.model
+    def create(self, vals):
+        attribute = super(ProductAttributeValue, self).create(vals)
+
+        if attribute.attribute_id.is_package:
+            (self.env['seed.package']
+                 .create(vals={'attribute_value_id': attribute.id}))
```
Comment (houssine78) [inline product_seeds/models/seed_package.py]: why puting "vals=" ? Just the dictionnary is enough

File: product_seeds/views/seed_package.xml [hunk 4]
```diff
@@ -0,0 +1,41 @@
...
+
+    <record id="view_seed_package_form" model="ir.ui.view">
+        <field name="name">seed.package.form</field>
+        <field name="model">seed.package</field>
+        <field name="arch" type="xml">
+            <form name="seed_package_form" string="Seed Package">
+                <group>
+                    <field name="name"/>
+                    <!--<field name="attribute_id"/>-->
+                    <!-- invisible -->
```
Comment (houssine78) [inline product_seeds/views/seed_package.xml]: remove commented code

**Review by houssine78 — APPROVED** [review]

File: product_seeds/data/data.xml (hunk 1 above)
Comment (robinkeunen) [reply product_seeds/data/data.xml:4]: I'll put a `fixme` tag and leave it like that while we're developing the module. It's gonna be `1` in the end.

**Pull request merged**